doc/api/fs.md

```diff
@@ -488,6 +488,16 @@ argument to `fs.createReadStream()`. If `path` is passed as a string, then
 `readStream.path` will be a string. If `path` is passed as a `Buffer`, then
 `readStream.path` will be a `Buffer`.
 
+### readStream.pending
```

Comment:
Would be better to name this `.isPending`. When you read `.pending`, it could either mean it contains a pending object or a boolean of whether its pending. That's not clear from just reading the code.

Reply:
It’s also what HTTP/2 already uses for this purpose…